src/Components/Results/Results.css

```diff
@@ -0,0 +1,63 @@
+.results {
+  background-color: rgb(20, 18, 17);
+  color: white;
+  display: flex;
+  flex-direction: column;
+}
+
+.results--loading, .results--none {
+  height: 100vh;
+  background-color: rgb(20, 18, 17);
+  color: white;
+  padding-top: 60px;
+}
+
+.results--none {
+  justify-content: center;
+  width: 100%;
+}
+
+.results--grid {
```

Comment:
this breaks around 300px wide. We should drop down to 1 album per line at smaller sizes or figure out how to deal with the white line on the right and the partially cut text on a 280px screen size.

Reply:
fixed!